lighting/models/product_flux.py

```diff
@@ -1,39 +0,0 @@
-# Copyright NuoBiT Solutions, S.L. (<https://www.nuobit.com>)
-# Eric Antones <eantones@nuobit.com>
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
-
-from odoo import api, fields, models, _
-from odoo.exceptions import UserError, ValidationError
-
-
-class LightingProductFlux(models.Model):
-    _name = 'lighting.product.flux'
```

Comment:
After updating the system, a recurring warning appears in the Odoo logs: WARNING odoo.modules.loading: Model 'lighting.product.flux' is declared but cannot be loaded! (Perhaps a module was partially removed or renamed). 
It seems that residual metadata has not been fully cleared.
There may still be data associated with this model in the database tables, such as ir_model and ir_model_data. Please conduct a comprehensive database search using the terms "lighting.product.flux" and "lighting_product_flux" to identify any lingering references. Once identified, ensure these are addressed in the migration script along with the DROP TABLE command.